orahlp.go

```diff
@@ -23,6 +23,12 @@ import (
 // Number as string
 type Number string
 
+// Timestamp as time.Time. For eJSON Marshal Support
+type Timestamp time.Time
```

Comment:
Where is it used?

Reply:
It was something added for eJSON , not needed now, removed it.